### ISSUE TITLE
Last in range day has incorrect border

### ISSUE THREAD

**Issue by krissalvador27** [airbnb/react-dates#1536]
**react-dates version**
react-dates@18.4.0

**Describe the bug**
Last in selected date range has wrong border top. In the screenshot below day 21 has a gray border (looks like it's the default border). Would expect it to be green like the days in the range before it.

**Source code (including props configuration)**
```
<DateRangePicker
  startDate={this.state.startDate}
  startDateId="your_unique_start_date_id"
  endDate={this.state.endDate}
  endDateId="your_unique_end_date_id"
  onDatesChange={({ startDate, endDate }) => this.setState({ startDate, endDate })}
  focusedInput={this.state.focusedInput}
  onFocusChange={focusedInput => this.setState({ focusedInput })}
/>
```

**Screenshots/Gifs**
<img width="596" alt="screen shot 2019-02-07 at 10 57 14 pm" src="https://user-images.githubusercontent.com/5139481/52458164-bd3d2580-2b2b-11e9-804f-2d2bad56155e.png">

**Desktop (please complete the following information):**
 - OS: Mac OS 10.14
 - Browser: Chrome 
 - Version 71.0.3578.98

**Is the issue reproducible in Storybook?**
http://airbnb.io/react-dates/?selectedKind=DateRangePicker%20%28DRP%29&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Additional context**
Inspecting the styles it looks like this rule 
```
.CalendarDay__last_in_range,
.CalendarDay__last_in_range:hover {
  border-style: solid;
}
```
conflicts with the default `border-style: double`.

**Comment by krissalvador27** [comment]
@ljharb would love to fix and test it, but I don't think the CSS file is exposed in the repo (only in the published version on npm). Any suggestions?

**Comment by ljharb** [comment]
The css file is built from the in-JS styles. You should be able to make a fix there.

**Comment by krissalvador27** [comment]
fixed in `v20.0.0`
